WBooks/Library/MainMenuController.swift

```diff
@@ -34,24 +35,12 @@ class MainMenuController: UIViewController {
         return false
     }
     
-    private func addBarButtons() {
-        let nav = self.navigationController?.navigationBar
-        self.title = "NAVIGATION_BAR_TITLE".localized()
-        nav?.titleTextAttributes = [NSAttributedStringKey.foregroundColor: UIColor.white]
-        //Left notification button
-        navigationItem.leftBarButtonItem = UIBarButtonItem.notificationButton()
-        //Right search button
-        navigationItem.rightBarButtonItem = UIBarButtonItem.searchButton()
-    }
-    
     private func configureTableView() {
         books = createArray()
         _view.tableView.delegate = self
         _view.tableView.dataSource = self
         _view.tableView.register(cell: BookCell.self)
-        _view.tableView.rowHeight = UITableViewAutomaticDimension
```

Comment:
Why you delete this?

Reply:
I have this code:
`func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
        return 100
    }`

Was that statement necessary? I checked and no changes came up on deletion.

Reply:
If you set `rowHeight` and `estimatedRowHeight` you don't need that function. But it's ok 👌